libvmi/os/windows/peparse.c

```diff
@@ -228,7 +233,7 @@ get_aon_index(
 
 status_t
 peparse_validate_pe_image(
-    const uint8_t * const image,
+    uint8_t *image,
```

Comment:
We ought to be able to keep the `const` qualifiers here, no?

Reply:
In https://github.com/libvmi/libvmi/pull/1072 you specifically asked for them to be removed, just keeping it consistent here.

Reply:
Not for the validate function, for the get function.

Reply:
In the get function the memory pointer to be image clearly gets modified, so no `const` qualifier should be used. Here, the memory read-only for the validation, so const can be used.